code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -353,6 +353,12 @@
 	if(C.suiciding)
 		return 0 //Kevorkian school of robotic medical assistants.
 
+	// is secretly a silicon
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		if(H.species && H.species.reagent_tag == PROCESS_SYN)
+			return 0
+
```

Comment:
Actually, you may want to do a check for `& PROCESS_ORG` instead, if we ever add more reagent tags (*scream).